src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -173,6 +173,9 @@ public static Calendar convertToUserTimeZone(Calendar time, double timeZone) {
      */
     @Deprecated
     public static Date convertLocalDateToUtc(Date localDate, double localTimeZone) {
+        if (localDate == null) {
+            return null;
+        }
```

Comment:
@leeyimin How about we do both? I could leave this here as a general case for any future calls to this helper method and I could also handle it in the getter methods in fsAttributes as the special case.